### PR TITLE
[master < T595-MAGE] Add remove label on c api

### PR DESCRIPTION
**Pull request by ind1xa**
[master < Task] PR
- [x] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [x] Write a release note here, including added/changed clauses: Users can now call C++ and C API to remove label.
- [x] Tag someone from docs team in the comments

### CODE REVIEW

File: include/mgp.hpp [hunk 1]
```diff
@@ -645,6 +645,8 @@ class Node {
   Relationships OutRelationships() const;
   /// @brief Adds a label to the node.
   void AddLabel(const std::string_view label);
+  /// @brief Removes a label from the node.
```
Comment (antoniofilipovic) [inline include/mgp.hpp:648]: Can you just add spacing here between functions. 
You can check this Ivan's PR: https://github.com/memgraph/memgraph/pull/1103/files

Comment (antoniofilipovic) [reply include/mgp.hpp:648]: Format also rest of the Relationships class so we are consistent

**Review by antoniofilipovic — APPROVED** [review]

**Comment by ind1xa** [comment]
@vpavicic RELEASE NOTE: Users can now call C++ and C API to remove label.

**Comment by vpavicic** [comment]
> @vpavicic RELEASE NOTE: Users can now call C++ and C API to remove label.

With vertex_remove_label() function?

**Comment by ind1xa** [comment]
> > @vpavicic RELEASE NOTE: Users can now call C++ and C API to remove label.
> 
> @ind1xa With vertex_remove_label() function?

@vpavicic and Node::RemoveLabel() for c++

**Comment by vpavicic** [comment]
Can I merge it like this:

The C++ API for writing custom query modules now enables removing properties from nodes with `RemoveProperty()` function and labels with `RemoveLabel()` function.

**Comment by ind1xa** [comment]
sure, just that I wasn't working on RemoveProperty(), that is Matijas PR I think

**Comment by vpavicic** [comment]
Of course, but users don't know (nor care :D )

**Pull request merged**